osbs/cli/main.py

```diff
@@ -718,6 +720,10 @@ def cli():
                               help='isolated build')
     build_parser.add_argument('--arrangement-version', action='store', required=False,
                               help='version of inner template to use')
+    build_parser.add_argument('--signing-intent', action='store', required=False,
+                              help='signing intent of the ODCS composes')
+    build_parser.add_argument("--compose-id", action='append', required=False,
+                              dest="compose_ids", help="ODCS compose used")
```

Comment:
How do you specify more than one? The help text should explain that.

Reply:
ok